packages/toolshed/routes/ai/llm/models.ts

```diff
@@ -73,9 +73,7 @@ const addModel = ({
     modelName = modelName.split("-thinking")[0];
   }
 
-  const model = providerOptions
-    ? provider(modelName, providerOptions)
-    : provider(modelName);
+  const model = provider(modelName);
```

Comment:
This looks like it might introduce bugs, but the API has changed and does not support this invocation anymore.